string effector_name
float32[] target_coordinate
---
